UnityProject/Assets/Scripts/Objects/Doors/Modules/AccessModule.cs

```diff
@@ -53,16 +53,19 @@ public override void ClosedInteraction(HandApply interaction, HashSet<DoorProces
 
 		public override void BumpingInteraction(GameObject byPlayer, HashSet<DoorProcessingStates> States)
 		{
-			if (!master.HasPower || !CheckAccess(byPlayer))
+			if (byPlayer != null)
 			{
-				States.Add(DoorProcessingStates.SoftwarePrevented);
+				if (!master.HasPower || !CheckAccess(byPlayer))
```

Comment:
bool thing again here (Know it was already like this, but still good to change).